modules_for_BSAT/fastq_analysis.py

```diff
@@ -0,0 +1,59 @@
+def is_nucleotide(seq: str) -> bool:
+    """
+    Function return 'True' if sequence is DNA or RNA.
+    
+    :param seq: some sequence
+    :type seq: str
+    :rtype: bool
+    :return: 'True' if sequence is DNA or RNA   
+    """
+    seq_alphabet = {'A', 'T', 'U', 'G', 'C'}
+    if set(seq.upper()) <= seq_alphabet:
+        return True
```

Comment:
Если эта проверка не будет пройдена, то вернется None, а лучше, чтобы возвращался False.
```suggestion
    if set(seq.upper()) <= seq_alphabet:
        return True
    return False
```